examples/dtls-echo/Makefile

```diff
@@ -24,51 +24,49 @@ BOARD_INSUFFICIENT_MEMORY := airfy-beacon b-l072z-lrwan1 bluepill calliope-mini
 # NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
 USEMODULE += gnrc_netdev_default
 USEMODULE += auto_init_gnrc_netif
-# Specify the mandatory networking modules for IPv6 and sUDP
+# Specify the mandatory networking modules for IPv6 and UDP
 USEMODULE += gnrc_ipv6_router_default
-USEMODULE += gnrc_udp
-# Add a routing protocol
+USEMODULE += gnrc_sock_udp
+# Add a routing protocol (optional)
```

Comment:
can you please remove any module not needed for the dtls example? IHMO, gnrc_rpl and icmpv6_echo are not required, even `ps` os not mandatory. Also I would simply use `gnrc_ipv6_default` instead of `_router_default`.

Keep it simple, if somebody wants to use multi-hop networking and run dtls over it, fine but I guess then they can also extend and add modules as needed.